supertokens_python/recipe/session/utils.py

```diff
@@ -326,6 +370,7 @@ def validate_and_normalise_user_input(
     error_handlers: Union[ErrorHandlers, None] = None,
     override: Union[InputOverrideConfig, None] = None,
     jwt: Union[JWTConfig, None] = None,
+    invalid_claim_status_code: Union[int, None] = None,
```

Comment:
The input config (from the user's point of view) also needs to change to contain this?

Reply:
Hmm. yes. Added that. 

Actually, shouldn't we provide this in override or something? Adding a new arg for this feels like an overkill.

Reply:
No. Not override. It should be an optional new argument.